dummy-relying-party/src/lib.rs

```diff
@@ -392,14 +392,6 @@ fn create_asset_response(
 ) -> HttpAssetResponse {
     // set up the default headers and include additional headers provided by the caller
     let mut headers = vec![
-        ("strict-transport-security".to_string(), "max-age=31536000; includeSubDomains".to_string()),
```

Comment:
This messed up the JS when deploying it.

I removed all the security headers because I didn't want to deal with any of these problems in a dummy project.

Reply:
Hm, we should investigate this to figure out _why_ you ran into issues.
I agree that we don't necessarily need security headers in the dummy project, but we need to check compatibility with actual production issuers that do need these headers.

Reply:
This was the relying party, not the issuer.

The security headers are independent of being relying party or not. Devs should follow the best practices for apps in general.